services/manage/app/controllers/manage/orders/edit/lineitems/edit/index.js

```diff
@@ -10,8 +11,15 @@ import lineitemPublishers from '@base-cms/parcel-plug-manage/gql/mutations/linei
 import lineitemDateDays from '@base-cms/parcel-plug-manage/gql/mutations/lineitem/date-days';
 import lineitemDateRange from '@base-cms/parcel-plug-manage/gql/mutations/lineitem/date-range';
 import deleteLineItem from '@base-cms/parcel-plug-manage/gql/mutations/lineitem/delete';
+import pauseLineItem from '@base-cms/parcel-plug-manage/gql/mutations/lineitem/pause';
 
 export default Controller.extend(ObjectQueryManager, ActionMixin, {
+  isActive: computed.not('model.paused'),
```

Comment:
There is not an `isActive()` method on the model schema... perhaps make this a graph field and use instead of a computed property